src/main/java/practice/StreamPractice.java

```diff
@@ -14,7 +18,13 @@ public class StreamPractice {
      * "Can't get min value from list: < Here is our input 'numbers' >"
      */
     public int findMinEvenNumber(List<String> numbers) {
-        return 0;
+        return numbers.stream()
+                .flatMap(s -> Arrays.stream(s.split(",")))
```

Comment:
For the future, do not use abbreviation names
```suggestion
                .flatMap(line -> Arrays.stream(line.split(SEPARATOR)))
```